app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala

```diff
@@ -187,6 +187,7 @@ class SubsidyController @Inject()(
               Ok(addClaimEoriPage(claimEoriForm.fill(OptionalEORI(a,x)), journey.previous))
             )
           }
+      case _ => handleMissingSessionData("Subsidy journey")
```

Comment:
NABD a lot of repeating here.

Reply:
Yeah it's to fix the warning about the `None` case not being matched in all these places. Needs another pass really to see if there's a better way to structure this, but this was a first step to just fix the warnings for now.